src/main/java/org/broadinstitute/dsm/model/participant/ParticipantWrapper.java

```diff
@@ -165,6 +170,25 @@ else if (DBConstants.DDP_ABSTRACTION_ALIAS.equals(source)) {
         }
     }
 
+    private List<String> getGovernedUserIdsFromElasticList(List<ElasticSearchParticipantDto> proxyData) {
+        List<String> participantGovernedUserIds = new ArrayList<>();
+
+        proxyData.stream()
+                .filter(esGovernedUserData -> !esGovernedUserData.getGovernedUsers().isEmpty())
+                .forEach(esGovernedUserData ->
+                        participantGovernedUserIds.addAll(esGovernedUserData.getGovernedUsers()));
+        return participantGovernedUserIds;
+    }
+
+    private List<String> combineListWithMatchingParticipantIds(List<String> participantIdsToFetch, Set<String> list) {
+        if (participantIdsToFetch.isEmpty()) {
+            return new ArrayList<>(list);
+        }
+        else {
+            return participantIdsToFetch.stream().filter(new HashSet<>(list)::contains).collect(Collectors.toList());
```

Comment:
is it necessary to wrap list into `HashSet` to use `contains` method? Maybe `filter(list::contains)` could be enough as well?